standardtracer/span.go

```diff
@@ -45,14 +56,14 @@ func (s *spanImpl) LogEventWithPayload(event string, payload interface{}) {
 }
 
 func (s *spanImpl) Log(ld opentracing.LogData) {
-	s.lock.Lock()
-	defer s.lock.Unlock()
+	s.Lock()
+	defer s.Unlock()
 
 	if ld.Timestamp.IsZero() {
 		ld.Timestamp = time.Now()
 	}
 
-	s.raw.Logs = append(s.raw.Logs, &ld)
+	s.raw.Logs = append(s.raw.Logs, ld)
```

Comment:
I suppose because LogData is a timestamp and 2 pointers, faster to copy it than to allocate on heap. I.e. the speed improvement is probably coming from the points where LogData is created.